lab_3_keywords_textrank/main.py

```diff
@@ -356,18 +418,22 @@ def __init__(self) -> None:
         """
         Constructs all the necessary attributes for the edge list graph object
         """
-        pass
+        self._edges = {}
+        self._positions = {}
+        self._position_weights = {}
+
 
     # Step 7.2
-    def get_vertices(self) -> tuple[int, ...]:
+    def get_vertices(self) -> tuple[str, ...]:
```

Comment:
why did you have to modify the typing annotation? 
it must stay the same as in the task description as it is a part of the task...

Reply:
asked to change because the return was highlighted